### DISCUSSION TITLE
Was würdet ihr bei 3 f) machen?

### DISCUSSION THREAD

**Discussion by samueltrippler** [malangmann/GitHub-Aufgabe_Gruppe1#3] [General]
In der Aufgabe steht ja: "(f) Eine Funktion, die eine geeignete Visualisierung von drei oder vier
kategorialen Variablen erstellt".

Was ist damit genau gemeint?

**Comment by malangmann** [comment]
Ein Mosaikplot ist denke ich auch noch eine Möglichkeit, zusätzlich zu der Umsetzung aktuell mit Balkendiagramm und Häufigkeiten.
